lib/jsdom/virtual-console.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+var VirtualConsole = require("events").EventEmitter;
```

Comment:
This doesn't work, because that means on the next line you're adding a new method to all EventEmitter instances. This could be most easily done with the `inheritFrom` util in `jsdom/utils.js`:

```js
function VirtualConsole() {}
inheritFrom(EventEmitter, VirtualConsole, {
  sendTo: function (anyConsole) { ... }
});
```